web/js/containers/info.js

```diff
@@ -191,18 +188,14 @@ const mapDispatchToProps = (dispatch) => ({
     }
   },
   aboutClick: () => {
-    if (util.browser.small) {
-      window.open('pages/about.html?v=@BUILD_NONCE@', '_blank');
-    } else {
-      dispatch(
-        requestTemplate(
-          ABOUT_PAGE_REQUEST,
-          'pages/about.html?v=@BUILD_NONCE@',
-          'text/html',
-        ),
-      );
-      dispatch(renderTemplate('About', 'modalAboutPage'));
-    }
+    // Create new functionality here that renders the about page
+    // inside a modal window.
+    dispatch(
+      openCustomContent('ABOUT_MODAL', {
+        headerText: 'About',
+        bodyComponent: AboutPage,
```

Comment:
You can use the `wrapClassName` property here to assign a class to the modal (this will allow you to avoid using element ids as CSS selectors).

Reply:
Great to know!